### PR TITLE
3758  - Add calendar and related docs

### LINKED ISSUE #3758: Calendar: Make docs live and improve them
**Is your feature request related to a problem or use case? Please describe.**
The calendar docs are not currently shown on https://design.infor.com/code/ids-enterprise/latest

**Describe the solution you'd like**
- Do a pass on calendar developer docs
- expose this on the design site

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by tmcconechy**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**

Exposes and adds Calendar, Monthview and WeekView Docs. Also fixed a bug i found.

ALSO: Pushed an update to cookies that was reccomended by our latest security scan for the demo app. To test this you can restart the server and test the kitchen sink page.

**Related github/jira issue (required)**:
Fixes #3758

**Steps necessary to review your pull request (required)**:
- open http://localhost:4000/components/calendar/example-only-calendar-legend.html
- should not show the mobile view below
- open http://localhost:4000/components/calendar/example-index
- resize to mobile should still change to the legend below
- you cant test these docs , but they should work just check for edits.

**Included in this Pull Request**:
- [x] A note to the change log.

### CODE REVIEW

File: src/components/week-view/readme.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+  "disabled": false
+},
+```
+
+The following data fields are used.
+
+- id - A unique eventType id
+- label - The label that will show up in the legend
+- translationKey - The key that will be looked up in the locale data
+- color - The color from the color pallete (without the color number)
```
Comment (jmacaluso711) [inline src/components/week-view/readme.md:87]: typo: "pallete" > "palette"

File: src/components/calendar/readme.md [hunk 2]
```diff
@@ -1,37 +1,44 @@
...
+The setup for a calendar only involves creating a `div` with the class `calendar`. In that div we add several sections for the calendar.
 
-The first pass of the calendar added basic single day event rendering, filtering and changing months.
-The second pass of the calendar added events that span months, icon support, ajax support, selection and month change events, and a readonly details template.
+- The `calendar-events` section contains an accordion with a filter section and an upcoming events section.
+- The `calendar-monthview` section is a `div` with where the monthview calendar will be rendered.
+- The `calendar-weekview` will hold the day and week container (if used)
+- The `calendar-event-details` is also an accordion where the event details will be shown when you click on a day.
+- The `calendar-event-mobile` shuold be a listview, which is the calendar-event-details section shown at mobile breakpoints as a list instead.
```
Comment (jmacaluso711) [inline src/components/calendar/readme.md:25]: typo: "shuold"

File: src/components/calendar/readme.md [hunk 3]
```diff
@@ -41,57 +48,91 @@ We will next add better responsiveness, better keyboard support, adding events a
...
-- location - a location for the event that will be shown in the event details
-- status - a text status that will be shown in the event details (with optional icon on the calendar)
-- starts - the date/time when the event starts
-- starts - the date/time when the event ends
-- type - the eventType key, this must have a match in the eventTypes array
-- isAllDay - determines if the event is all day or not
+- id - A unique eventType id
+- label - The label that will show up in the legend
+- translationKey - The key that will be looked up in the locale data
+- color - The color from the color pallete (without the color number)
```
Comment (jmacaluso711) [inline src/components/calendar/readme.md:110]: typo: "pallete" > "palette"

**Comment by tmcconechy** [comment]
Fixed @jmacaluso711 also in other places...

**Pull request merged**